### PR TITLE
Starting to debug diff drive controller in sim.

### PR DESCRIPTION
**Pull request by destogl**
ros-simulation/gazebo_ros2_control#97 - we have to sync with this

### CODE REVIEW

File: diff_drive_controller/src/diff_drive_controller.cpp [hunk 1]
```diff
@@ -157,13 +156,20 @@ controller_interface::return_type DiffDriveController::update(
...
   }
 
-  const auto age_of_last_command = current_time - last_command_msg->header.stamp;
-  // Brake if cmd_vel has timeout, override the stored command
-  if (age_of_last_command > cmd_vel_timeout_)
-  {
-    last_command_msg->twist.linear.x = 0.0;
-    last_command_msg->twist.angular.z = 0.0;
-  }
+//   if (last_command_msg->header.stamp.sec != 0 and last_command_msg->header.stamp.nanosec != 0)
```
Comment (bmagyar) [inline diff_drive_controller/src/diff_drive_controller.cpp:159]: I think we discussed this enough but basically there should be a common policy on this and we should offer only one solution. An optional addition can be to opt-out of this feature altogether, in that case one could publish commands with timestamps from different time sources, we wouldn't know.

**Comment by destogl** [comment]
Obsolete.

**Comment by Raamkishore** [comment]
The changes are not reflected in the main repository and its branches.

**Comment by Raamkishore** [comment]
I guess the changes are not updated in all the branches. I cloned the galactic branch and it took me a good amount of time to realise the problem with just the diff drive controller.

**Comment by bmagyar** [comment]
Hence why this is still an open PR

**Pull request closed without merging**